validation_wrapper.py

```diff
@@ -9,9 +9,10 @@
 import traceback
```

Comment:
probably also an old mathias script that is not used for anything

Reply:
Will fix in separate PR